### PR TITLE
x27lu montreal

### LINKED ISSUE #3: x27lu's checklist
- [x] Test
- [ ] Work Locally
- [ ] Pull request
- [ ] Merge

### PR DESCRIPTION
**Pull request by x27lu**
Restaurant suggestions @githubteacher please review.

Closes #3

### CODE REVIEW

File: x27lu-montreal.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Montreal, Ontario, Canada
+
+- Schwartz's
+- Schwartz's
+- Schwartz's
+
+# Toronto, Ontario, Canada
+
+1. La Carnita
```
Comment (githubteacher) [inline x27lu-montreal.md]: Is this where I can get solid 🌮 s?

File: x27lu-montreal.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Montreal, Ontario, Canada
+
+- Schwartz's
+- Schwartz's
```
Comment (githubteacher) [inline x27lu-montreal.md:4]: What should I get here?

**Review by githubteacher — COMMENTED** [review]
just some questions about the content.

File: x27lu-montreal.md (hunk 2 above)
Comment (x27lu) [reply x27lu-montreal.md:4]: Montreal Smoked Meat sandwich is their classic

File: x27lu-montreal.md (hunk 1 above)
Comment (x27lu) [reply x27lu-montreal.md]: Yup

File: x27lu-montreal.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+- Schwartz's
+
```
Comment (hxietest) [inline x27lu-montreal.md]: A comment from Hill

**Review by hxietest — COMMENTED** [review]
Review summary from Hill

**Review by hxietest — APPROVED** [review]
Approved

**Pull request merged**